src/charm.py

```diff
@@ -127,11 +150,14 @@ def _get_smtp_data(self) -> smtp.SmtpRelationData:
         Returns:
             SmtpRelationData containing the SMTP details.
         """
+        peer_relation = self.model.get_relation("smtp-peers")
+        assert peer_relation  # nosec
```

Comment:
I'm wondering if `# nosec` is obvious or not to have a comment explaining the reason for it - does peer_relation exist in a single unit deployment?

Reply:
I think it quite obvious, and you?

Reply:
okay, i wasn't sure if peer relation is setup in single unit deployment, now I know so I guess there's no need :D